log/development.log

```diff
@@ -80,3 +80,58 @@ FOREIGN KEY ("post_id")
   [1m[36mTRANSACTION (0.8ms)[0m  [1m[35mCOMMIT[0m
   [1m[35m (0.3ms)[0m  [1m[34mSELECT pg_advisory_unlock(1365696269427442995)[0m
   [1m[36mActiveRecord::SchemaMigration Pluck (0.6ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
+Started GET "/" for ::1 at 2022-12-05 18:41:39 +0530
```

Comment:
[OPTIONAL] Great description, I liked the structure. Please add your own words in the PR description. About what you did in this project and highlights. 😉

<img width="848" alt="Screenshot 2022-12-05 at 18 40 39" src="https://user-images.githubusercontent.com/69011963/205651735-3b3e1fcb-2b62-4f27-b3af-153d83d3de9a.png">